components/common/Footer.tsx

```diff
@@ -42,6 +43,14 @@ const Footer = () => (
           <GitHubIcon />
           <span>Github Link</span>
         </a>
+        <a
+          href="https://discord.gg/8MfNdGK"
```

Comment:
I assume we can change this to a Vanity URL instead. We got one months after this PR was made.

Reply:
It's https://discord.gg/foss

Reply:
Also, Netwise is no longer a sponsor